app/scripts/modules/openstack/serverGroup/configure/wizard/access/accessSettings.html

```diff
@@ -28,6 +28,36 @@
                                             required="true"
                                             model="command.securityGroups"></os-app-cache-backed-multi-select-field>
 
+    <div class="form-group">
+      <div class="col-md-5 sm-label-left">
+        <b>Associate Floating IP Address</b>
+        <help-field key="openstack.network.floatingip"></help-field>
+      </div>
+      <div class="col-md-2 radio">
+        <label>
+          <input type="radio" ng-model="command.associatePublicIpAddress" ng-value="true" id="associatePublicIpAddressTrue"/>
+          Yes
+        </label>
+      </div>
+      <div class="col-md-2 radio">
+        <label>
+          <input type="radio" ng-model="command.associatePublicIpAddress" ng-value="false" id="associatePublicIpAddressFalse"/>
+          No
+        </label>
+      </div>
+      <div class="col-md-2 radio">
+        <label>
+          <input type="radio" ng-model="command.associatePublicIpAddress" ng-value="null" id="associatePublicIpAddressDefault"/>
+          Default
+        </label>
+      </div>
+    </div>
+
+    <div ng-if="command.associatePublicIpAddress == true" class="form-group">
```

Comment:
Please use `===` here -- there won't be any problems as it is written, but it looks dangerous, since the statement will be true for any `command.associatePublicIpAddress` that is JS true.